spyder/widgets/projects/projectdialog.py

```diff
@@ -152,8 +152,9 @@ def _get_project_types(self):
 
     def select_location(self):
         """Select directory."""
-        location = getexistingdirectory(self, _("Select directory"),
-                                        self.location)
+        location = osp.normpath(getexistingdirectory(self, _("Select directory"), 

```

Comment:
Hi @vitoriahmc please remove the trailing whitespace at the end of this line. You can check if you are following PEP8 with the comment made by [pep8speaks](https://github.com/spyder-ide/spyder/pull/7698#issuecomment-413328101)